tests/test_1.py

```diff
@@ -52,37 +52,32 @@
 }
 
 
-def castles_number(heights):
-    start_height, current_height = None, None
-    castles = 0
-
+def get_castles_number_1(heights):
     if len(heights) == 0:
-        raise Exception
+        raise Exception('Border is empty')
     if len(heights) == 1:
         return 1
 
+    castles = 0
+    current_height = heights[0]
+
     for i in range(len(heights)):
```

Comment:
If `current_height = heights[0]` why do you need start the `for` loop with `0`?
If you are starting with `0` then first execution of the body of the loop always has no sense.